server/main.py

```diff
@@ -238,19 +261,19 @@ async def room(websocket: WebSocket) -> None:
     try:
         while True:
             event = await client.receive()
+            if event is None:
+                return
```

Comment:
We might want to continue if the event is `None`.

Reply:
In what case? If event is None, then an error was raised

Reply:
> In what case? If event is None, then an error was raised

Yeah, but if, for example, the data was invalid, we don't want to disconnect the client, but just ignore the request and continue receiving events.